test/k8sT/manifests/microscope.yaml

```diff
@@ -72,3 +72,13 @@ spec:
     image: docker.io/cilium/microscope:1.1.0-ci
     imagePullPolicy: IfNotPresent
     name: microscope
+    readinessProbe:
+      exec:
+        command:
+        - microscope
+        - --send-command
+        - "'echo test'"
+        - --timeout-monitors
+        - "5"
+      initialDelaySeconds: 5
```

Comment:
Is this needed? I have concerns about this one, it'll be always installed per each test, so we're going to add 3 min to the test, if we can avoid will be great.

Reply:
It takes from 10s to one minute from my testing for RBAC rules to kick in. We can add an issue to track running microscope pod only once.